tests/testthat/test-bump_version.R

```diff
@@ -1,14 +1,12 @@
 test_that("bump_version() works", {
-  skip_if_not_installed("rlang", "0.4.11.9001")
-
   tempdir <- withr::local_tempdir(pattern = "fledge")
   repo <- create_demo_project(open = FALSE, dir = tempdir, news = TRUE)
   usethis::with_project(
     path = repo, {
       use_r("bla")
       gert::git_add("R/bla.R")
       gert::git_commit("* Add cool bla.")
-      expect_snapshot(bump_version())
+      expect_snapshot(bump_version(), variant = as.character(packageVersion("rlang")))
```

Comment:
Nice idea, it seems that using the version directly is not sustainable. Maybe something like:

```suggestion
      expect_snapshot(bump_version(), variant = if (packageVersion("rlang") > "0.99") "rlang-1" else "rlang-legacy")
```

Maybe refactor out the rlang variant to a function?

Reply:
Done!